src/coinbase/wallet.ts

```diff
@@ -755,6 +756,22 @@ export class Wallet {
     return await this.getDefaultAddress()!.createTransfer(options);
   }
 
+  /**
+   * Creates a Payload Signature.
+   *
+   * @param unsignedPayload - The Unisgned Payload to sign.
```

Comment:
Unsigned

Reply:
Fixed in most recent commit! cc: @jazz-cb